lumeer-core/src/main/java/io/lumeer/core/facade/TemplateFacade.java

```diff
@@ -74,16 +78,34 @@ public String getTemplateOrganizationId(final Language language) {
       }
    }
 
+   private List<String> getAllTemplateOrganizationIds() {
+      final List<String> result = new ArrayList<>();
+
+      final String csOrg = defaultConfigurationProducer.get(DefaultConfigurationProducer.TEMPLATE_ORG_CS);
+      final String enOrg = defaultConfigurationProducer.get(DefaultConfigurationProducer.TEMPLATE_ORG_EN);
+
+      if (csOrg != null) {
+         result.add(csOrg);
+      }
+
+      if (enOrg != null) {
+         result.add(enOrg);
+      }
+
+      return result;
+   }
+
    public void installTemplate(final Project project, final String templateType, final Language language) {
       final TemplateParser templateParser = new TemplateParser(templateType, language);
 
-      installTemplate(project, templateParser, createTemplateMetadata(new Date()));
+      installTemplate(project, templateParser, createTemplateMetadata(new Date()), true);
    }
 
-   public void installTemplate(final Project project, final ProjectContent projectContent, final Date relativeDate) {
+   public void installTemplate(final Project project, final String organizationId, final ProjectContent projectContent, final Date relativeDate) {
       final TemplateParser templateParser = new TemplateParser(projectContent);
+      final boolean originalLumeerTemplate = getAllTemplateOrganizationIds().contains(organizationId);
```

Comment:
I think this code is not needed, because in `CopyFacade` we can easily distinguish between Template creation and Copying project